byzcoin/service.go

```diff
@@ -2656,6 +2718,7 @@ func newService(c *onet.Context) (onet.Service, error) {
 		closed:                 true,
 		catchingUpHistory:      make(map[string]time.Time),
 		rotationWindow:         defaultRotationWindow,
+		txErrorBuf:             newRingBuf(10),
```

Comment:
This should be big enough to keep at least all errors from all transactions from two blocks. If we suppose that all transactions create errors, for a 1MB block with 1kB transactions, that is 1024 transactions. For two blocks (just to be sure in case `AddTransaction` is behind one block), this makes 2048 errors.

```suggestion
		txErrorBuf:             newRingBuf(2048),
```